server/views/pages/view/cases.njk

```diff
@@ -22,13 +26,25 @@
     {% else %}
         {% set comDetailsHtml = 'Unallocated' %}
     {% endif %}
+
+    {% if case.licenceVersionOf %}
```

Comment:
Will this not mean that if you have an approved edit (eg v1.0 is INACTIVE and v1.1 is APPROVED) there will be a link to v1.1 to print the last approved licence, as v1.1 will have a `versionOf` pointing to v1.0?

Reply:
good spot, think you are right will check

Reply:
fixed, so that we only display the link for in progress or submitted versions